packages/cloud_controller_ng/pre_packaging

```diff
@@ -2,13 +2,20 @@ set -e -x
 
 cd ${BUILD_DIR}/cloud_controller_ng
 
+input_file="Gemfile"
+file_content=$(<"$input_file")
+modified_content=$(echo "$file_content" | sed '/^group :development do/,/^end/ d' | sed '/^group :test do/,/^end/ d')
+echo "$modified_content" > "$input_file"
+
 rm -rf ./vendor/cache/
 
 set +e
 for i in {1..3}; do
   bundle config --local specific_platform true
   bundle config --local path ./vendor/cache
   bundle config set cache_all true
+  bundle config set --local without 'development test'
```

Comment:
I think this config is not needed anymore, as the development and test gems have now been removed from the Gemfile.

Reply:
Ah yes that's true